src/NUnitEngine/nunit.engine/RuntimeFramework.cs

```diff
@@ -53,6 +53,19 @@ public enum RuntimeType
     }
 
     /// <summary>
+    /// Enumeration identifying profile of .NET Framework
+    /// </summary>
+    public enum Profile
+    {
+        /// <summary>Profile Unspecified</summary>
+        Unspecified,
+        /// <summary>Client Profile</summary>
+        Client,
+        /// <summary>Full Profile</summary>
+        Full
+    }
```

Comment:
Should this be an enum or a simple string? In the MS class FrameworkName, it's just a string. In theory, this allows us to add any new profiles that come up more easily. Of course, we have to be aware that client is a subset of full and that it only exists in certain versions.

Also, I wonder if Unspecified should actually mean Full?

@rprouse What do you think?